src/mcp/server/session.py

```diff
@@ -244,10 +245,47 @@ async def create_message(
         stop_sequences: list[str] | None = None,
         metadata: dict[str, Any] | None = None,
         model_preferences: types.ModelPreferences | None = None,
-        tools: list[types.Tool] | None = None,
```

Comment:
note: this confused me (why not leave it out?), but its a kwarg, and is needed to discriminate between the overloads